pkg/collector/corechecks/servicediscovery/module/ignore_proc.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
why is it in a new file?

Reply:
source file impl_linux.go already reached 600 lines, which I think it considered large file, I didn't want to increase this file further, this is why new file.

Reply:
it is misleading to have methods of the same struct defined on multiple files
let's fold it back to the original file

Reply:
done